apps/todo/angular/src/app/todo/todos/todos.component.ts

```diff
@@ -1,9 +1,16 @@
 import { Component, Input, Output, EventEmitter } from '@angular/core';
 import { TodoVM } from '@nx-clean/todo-presentation';
+import { TodoItemComponent } from '../todo-item/todo-item.component';
+import { NgFor } from '@angular/common';
 
 @Component({
-  selector: 'todo-todos',
-  templateUrl: './todos.component.html',
+    selector: 'todo-todos',
+    templateUrl: './todos.component.html',
+    standalone: true,
```

Comment:
why are the components standalone?

Reply:
Make sense to me because right now we are not using any more modules, getting rid of material modules, so makes more simple in that way to show the example and use a modern way that the framework angular goes, also in terms of performance is little bit better doing the compilation and tree shaking to have lighter bundler sizes using the stand alone api

Reply:
ok, it makes sense